### PR TITLE
remove the incorrect description

### PR DESCRIPTION
**Pull request by payiAzure**

### CODE REVIEW

File: api-reference/v1.0/resources/servicehealthissue.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Inherits from [serviceAnnouncementBase](../resources/serviceannouncementbase.md)
...
 |investigationSuspended|If our detailed investigation of a potential issue results in a request for additional information from customers to allow the service team to investigate further, you'll see this status. If service team need you to act, they'll let you know what data or logs they need.|
-|resolved|The Microsoft Windows service status that corresponds to service restored.|
-|mitigatedExternal|The Windows service status that corresponds to restoring service.|
-|mitigated|The Windows service status that corresponds to extended recovery.|
-|resolvedExternal|The Windows service status that corresponds to investigation suspended.|
-|confirmed|The Windows service status that corresponds to service interruption.|
-|reported|The Windows service status that corresponds to investigating.|
+|resolved|The health status reserved for future use.|
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/resources/servicehealthissue.md:62]: Lines 62-67:
@payiAzure , I suppose these were exposed unintentionally?
Can you change their descriptions to just "Reserved for future use."?

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
Hi @payiAzure , one comment that applies to servicehealthissue.md in beta and in v1. 
After you incorporate the feedback, the PR is good for publishing. Thanks.

File: api-reference/beta/resources/servicehealthissue.md [hunk 2]
```diff
@@ -61,12 +61,12 @@ Inherits from [serviceAnnouncementBase](../resources/serviceannouncementbase.md)
...
 |investigationSuspended|If our detailed investigation of a potential issue results in a request for additional information from customers to allow the service team to investigate further, you'll see this status. If service team need you to act, they'll let you know what data or logs they need.|
-|resolved|The Microsoft Windows service status that corresponds to service restored.|
-|mitigatedExternal|The Windows service status that corresponds to restoring service.|
-|mitigated|The Windows service status that corresponds to extended recovery.|
-|resolvedExternal|The Windows service status that corresponds to investigation suspended.|
-|confirmed|The Windows service status that corresponds to service interruption.|
-|reported|The Windows service status that corresponds to investigating.|
+|resolved|The health status reserved for future use.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/servicehealthissue.md:64]: Lines 64-69:
@payiAzure , can you change their descriptions to "Reserved for future use."?

File: api-reference/v1.0/resources/servicehealthissue.md (hunk 1 above)
Comment (payiAzure) [reply api-reference/v1.0/resources/servicehealthissue.md:62]: Done

File: api-reference/beta/resources/servicehealthissue.md (hunk 2 above)
Comment (payiAzure) [reply api-reference/beta/resources/servicehealthissue.md:64]: Done

**Review by angelgolfer-ms — APPROVED** [review]
Thanks for the details in email, and for incorporating my previous comments.

**Pull request merged**